src/components/PaddedContainer/index.tsx

```diff
@@ -32,10 +32,10 @@ interface PaddedContainerProps {
 
 const StyledContainer = styled('div')<PaddedContainerProps>(
   ({ top, left, bottom, right, gutter }) => css`
-    ${top ? `padding-top: ${distance[top]};` : ''}
-    ${left ? `padding-left: ${distance[left]};` : ''}
-    ${bottom ? `padding-bottom: ${distance[bottom]};` : ''}
-    ${right ? `padding-right: ${distance[right]};` : ''}
+    padding-top: ${(top && distance[top]) || 0};
+    padding-left: ${(left && distance[left]) || 0};
+    padding-bottom: ${(bottom && distance[bottom]) || 0};
+    padding-right: ${(right && distance[right]) || 0};
```

Comment:
why these changes are needed?

Reply:
to forcibly set padding to `0px`

Reply:
Thanks, I know what `padding` css property means. The question was why do you need to set it forcibly?

Reply:
To override default props if it is existing. For instance, take a look at https://github.com/ZEISS/precise-ui/pull/139/files?file-filters%5B%5D=.md&file-filters%5B%5D=.png&file-filters%5B%5D=.tsx#diff-98434080865c9ea31f591a968fbe9909

By default `InputInfo` has `padding-left="medium"` and now it is possible to  forcibly reset this value.